pkg/vault_cli.go

```diff
@@ -40,9 +40,13 @@ func (v VaultCli) dir() string {
 	return filepath.Join(v.aem.baseOpts.ToolDir, "vault-cli")
 }
 
-func (v VaultCli) execDir() string {
+func (v VaultCli) execFile() string {
 	vaultDir, _, _ := strings.Cut(filepath.Base(v.DownloadURL), "-bin")
-	return filepath.Join(v.dir(), vaultDir, "bin")
+	execDir := filepath.Join(v.dir(), vaultDir, "bin")
+	if osx.IsWindows() {
```

Comment:
have you checked this on windows?

Reply:
be aware of such quirks... https://github.com/wttech/aemc/blob/main/pkg/local_instance.go#L809-L810